src/classes/COFF_Affiliation_TDTM.cls

```diff
@@ -471,20 +489,20 @@ public class COFF_Affiliation_TDTM extends TDTM_Runnable {
     }
 
    /*******************************************************************************************************
-    * @description Determines whether an Affiliation to the Department the Course Offering belongs to already
-    * exists for each faculty member. Public only so it can be called from the tests.
+    * @description Determines whether an Affiliation to the Account the Course Offering belongs to already
```

Comment:
I kept the variable name when the method is used the same but changed the method to make it more generic.